src/System.IO.Ports/tests/Configurations.props

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard-Windows_NT;
-      netstandard;
       netfx;
     </BuildConfigurations>
```

Comment:
Hmm, the `netstandard-Windows_NT` should run core tests, and `netfx` run desktop (aka netfx) tests. Let's see what happens.